aten/src/ATen/native/cuda/DistributionTemplates.h

```diff
@@ -254,6 +254,9 @@ void distribution_binary_kernel(TensorIterator &iter, PhiloxCudaState philox_arg
   const input_t_1 *input_data_1 = static_cast<const input_t_1 *>(iter.data_ptr(1));
   const input_t_2 *input_data_2 = static_cast<const input_t_2 *>(iter.data_ptr(2));
 
+  const int num_threads = at::cuda::warp_size() * 2;
```

Comment:
In https://github.com/pytorch/pytorch/pull/68008, this value is changed to warpsize * 4